### PR TITLE
#42 SELECT WHERE IsSqlNull with all Field Types

### PR DESCRIPTION
**Pull request by Manish-Celestial**
This pull request is related to #42 

In the first iteration of the PR, all newly added fields along with updated list and tables has been committed

### CODE REVIEW

**Comment by Manish-Celestial** [comment]
Hi @robe070 , I have updated the relevant methods in the test case driver to test *sqlNUll and *sqlNotNull on all the field types.
Attaching a snapshot of DB and verifiertestreport of succesful execution 

image
[Verifier_Test_Report.txt](https://github.com/lansa/db-regression/files/13898734/Verifier_Test_Report.txt)

File: Partition (WBP)/Functions/V56118C.VT156118.fun.yml [hunk 1]
```diff
@@ -220,23 +265,33 @@
...
-    *
-    Mthroutine Name(mthSelect_Where_IsNotSqlnull)
-    Define_Map For(*INPUT) Class(#F156118K1) Name(#this_key)
-    Define_Map For(*INPUT) Class(#Std_Count) Name(#this_expected_number_of_records)
-    Define_Map For(*BOTH) Class(#BIFA002A) Name(#this_rc)
+    * Boolean Column.
+    If (#this_rc <> ER)
+    #Com_Owner.mthAdd_SubMsg( OK '- ((#F156118K2 <> 0) *And (#BIFB001C.IsSqlnull))' '' )
+    #LF_i := 0
+    Select Fields(*ALL) From_File(VTL156118C) Where((#F156118K2 <> 0) *And (#F156118D.IsSqlnull)) With_Key(#this_key) Io_Status(#this_rc) Io_Error(*NEXT) Val_Error(*NEXT)
```
Comment (robe070) [inline Partition (WBP)/Functions/V56118C.VT156118.fun.yml]: You have not changed the WHERE clause in ALL of the SELECT commands. E.g. for this line the WHERE parameter should be:
```
(#F156118K2 <> 0) *And (#BIFB001C.IsSqlnull)
```
I note that the log message before the SELECT and the IF command to test the returned value are using the correct Field.
Please correct all the SELECT commands.

**Review by robe070 — CHANGES_REQUESTED** [review]
Its good to see that you found other examples of initialising LOBs and Binary fields.

Also you'll need to modify verifier_connection.dat to enable the SQL Server and IBMi SuperServer tests. And also deliver all the object changes to the IBMi before you may run the test.

**Comment by Manish-Celestial** [comment]
Pushed a new commit, making the changes as needed

**Review by robe070 — COMMENTED** [review]
Changes look good.
Now just need the final test report with all tests run including Superserver and IBMi.

**Pull request merged**